make rpm - produces ansible_test inside the package

##### SUMMARY
`make rpm` produces an rpm package which only has `ansible_tests` python package inside
and thus cannot be installed.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
ansible binaries

##### ANSIBLE VERSION
latest checkout from development branch

##### CONFIGURATION
Nothing special.

##### OS / ENVIRONMENT
latest CentOS

##### STEPS TO REPRODUCE
```shell
git checkout https://github.com/ansible/ansible.git
cd ansible
make rpm
```

then list the contents of rpm-build/ansible-*****.devel.el7.noarch.rpm
```rpm -qpl rpm-build/ansible-*****.devel.el7.noarch.rpm```

##### EXPECTED RESULTS
It should contain ansible package.


##### ACTUAL RESULTS
It only contains ansible_tests

```paste below
 rpm-build]# rpm -qlp ansible-2.10.0-100.git201909172030.d3ec5ca.devel.el7.noarch.rpm
/usr/bin/ansible-test
/usr/lib/python2.7/site-packages/ansible_test
/usr/lib/python2.7/site-packages/ansible_test/__init__.py
/usr/lib/python2.7/site-packages/ansible_test/__init__.pyc
/usr/lib/python2.7/site-packages/ansible_test/__init__.pyo
/usr/lib/python2.7/site-packages/ansible_test/_data
/usr/lib/python2.7/site-packages/ansible_test/_data/ansible.cfg
/usr/lib/python2.7/site-packages/ansible_test/_data/cli
...
/usr/lib/python2.7/site-packages/ansible_test/config/cloud-config-vultr.ini.template
/usr/lib/python2.7/site-packages/ansible_test/config/inventory.networking.template
/usr/lib/python2.7/site-packages/ansible_test/config/inventory.winrm.template
```

so` rpm -ivh <package_name>` fails ...
```
# yum -y install ansible-2.10.0-100.git201909201522.bb0fa0a.devel.el7.noarch.rpm
Loaded plugins: fastestmirror
Examining ansible-2.10.0-100.git201909201522.bb0fa0a.devel.el7.noarch.rpm: ansible-test-2.10.0-100.git201909201522.bb0fa0a.devel.el7.noarch
Marking ansible-2.10.0-100.git201909201522.bb0fa0a.devel.el7.noarch.rpm to be installed
Resolving Dependencies
--> Running transaction check
---> Package ansible-test.noarch 0:2.10.0-100.git201909201522.bb0fa0a.devel.el7 will be installed
--> Processing Dependency: ansible = 2.10.0-100.git201909201522.bb0fa0a.devel.el7 for package: ansible-test-2.10.0-100.git201909201522.bb0fa0a.devel.el7.noarch
Loading mirror speeds from cached hostfile
* base: repo.uk.bigstepcloud.com
 * epel: epel.mirror.it2go.eu
 * extras: repo.uk.bigstepcloud.com
 * updates: mirrors.ukfast.co.uk
--> Finished Dependency Resolution
Error: Package: ansible-test-2.10.0-100.git201909201522.bb0fa0a.devel.el7.noarch (/ansible-2.10.0-100.git201909201522.bb0fa0a.devel.el7.noarch)
           Requires: ansible = 2.10.0-100.git201909201522.bb0fa0a.devel.el7
           Available: ansible-2.4.2.0-2.el7.noarch (extras)
               ansible = 2.4.2.0-2.el7
           Available: ansible-2.8.4-1.el7.noarch (epel)
               ansible = 2.8.4-1.el7
 You could try using --skip-broken to work around the problem
You could try running: rpm -Va --nofiles --nodigest
make: *** [install] Error 1
script returned exit code 2
```

--- a/Makefile
+++ b/Makefile
@@ -301,7 +301,7 @@ rpm: rpmcommon
 	--define "_srcrpmdir %{_topdir}" \
 	--define "_specdir $(RPMSPECDIR)" \
 	--define "_sourcedir %{_topdir}" \
-	--define "_rpmfilename $(RPMNVR).%%{ARCH}.rpm" \
+	--define "_rpmfilename %%{NAME}-%%{VERSION}-%%{RELEASE}.%%{ARCH}.rpm" \
 	--define "__python `which $(PYTHON)`" \
 	--define "upstream_version $(VERSION)" \
 	--define "rpmversion $(RPMVERSION)" \

